src/env.h

```diff
@@ -165,6 +165,7 @@ namespace node {
   V(priority_string, "priority")                                              \
   V(processed_string, "processed")                                            \
   V(prototype_string, "prototype")                                            \
+  V(public_key_string, "publicKey")                                           \
```

Comment:
I agree, `subjectPublicKeyInfo` is a much better name for this because that's what we actually want here.